packages/editor-ui/src/constants.ts

```diff
@@ -1,3 +1,5 @@
+export const MAX_WORKFLOW_SIZE = 16777216;
+export const MAX_WORKFLOW_PINNED_DATA_SIZE = 12582912;
```

Comment:
Let's add the unit in the const name or as a comment.

Reply:
Done.